cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -136,14 +136,15 @@ extends ColumnStore with CassandraChunkSource with StrictLogging {
     }
   }
 
-  private def writeIndices(dataset: Dataset,
-                           partition: Array[Byte],
-                           chunkset: ChunkSet,
-                           diskTimeToLive: Int): Future[Response] = {
-    asyncSubtrace("write-index", "ingestion") {
-      val indexTable = getOrCreateIndexTable(dataset.ref)
-      val indices = Seq((chunkset.info.id, ChunkSetInfo.toBytes(chunkset.info)))
-      indexTable.writeIndices(partition, indices, sinkStats, diskTimeToLive)
+  private def writeIngestion(dataset: Dataset,
+                             partition: Array[Byte],
+                             chunkset: ChunkSet,
+                             diskTimeToLive: Int): Future[Response] = {
+    asyncSubtrace("write-ingestion", "ingestion") {
+      val ingestionTable = getOrCreateIngestionTable(dataset.ref)
+      val info = chunkset.info
+      val infos = Seq((info.ingestionTime, info.startTime, ChunkSetInfo.toBytes(info)))
+      ingestionTable.writeIngestion(partition, infos, sinkStats, diskTimeToLive)
```

Comment:
Any repair based on ingestion time would need this, and the storage overhead isn't that much. A separate TTL just makes things more complicated and doesn't offer much benefit, in my opinion.

Reply:
Ah yes - downsampled records would indeed need it for repair. I forgot about that. But would we ever repair data more than a week old? Would reverse index be needed for old data?

I thought we discussed about having a lower TTL for this. What changed since that time ?

Reply:
For the time being, ODP queries fetch records from the ingestion index table first, and so those records must live at least as long as the records they refer to. Otherwise, old data cannot be queried at all.

Reply:
Agreed. I only ask that there be a separate parameter for the IngestionTimeRecord TTL. The values fed for both can be the same.

Reply:
It's trivial to change later when the need arises. I cannot see why it makes any difference, and adding another parameter just makes things a little more complicated with no clear benefit. I'd have to modify the ChunkSink trait too. If more secondary indexes are added, does each one require a separate TTL? This is a leaky abstraction.

Reply:
The idea was just to save storage space in Cassandra. We discussed that we didn't need to keep this around for 6 months, possibly more when it increases. I was pushing for it since I would be needing it when I write  downsampled data, and would pass separate lower TTL value there. 

Anyway, I defer to you to make this call. 

That said, I see your point on the leaky abstractions. This config value would be part of the cassandra config  passed into the impl constructor, and not be included in the Store interface.

Reply:
The discussion regarding the TTL was before I understood exactly how the index table was being used. Replacing the index table and changing the TTL can introduce new problems that don't exist today. Until we can safely say that the index has no purpose other than doing repairs, it's safest to keep the TTLs consistent among the two chunk tables.

Reply:
Also, this change is disruptive enough that I don't think its a good idea to make more changes than is necessary. A later PR can address new features and optimizations.